wiki/People/Nomination_Assessment_Team/Evaluations/en.md

```diff
@@ -22,6 +22,10 @@ For full Beatmap Nominators without recent warnings, their evaluations will be s
 
 The duration between evaluations is automatically scheduled by the [BN website](https://bn.mappersguild.com/). However, if there is a substantial cause for concern about a BN's performance at a sooner date, NAT members can also manually schedule an evaluation to happen sooner than normal.
 
+A Nominator is able to see the current progress of their evaluation via a progress bar in their BN website profile.
```

Comment:
is there a particular reason Nominator is used here instead of Beatmap Nominator? if its including NAT members then i think it should be clarified, nominators is a confusing term to encapsulate both BNs and NATs.

Reply:
```suggestion
A BN is able to see the current progress of their evaluation via a progress bar in their BN website profile.
```